Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,27 @@
 import React from 'react';
 import './Card.css';
+import '../../App.css';
```

Comment:
You don't have to include the css file in every file. This file was already included in App.js